tests/UnifiedSpecTests/versioned-api/crud-api-version-1-strict.json

```diff
@@ -0,0 +1,1115 @@
+          "events": [
+            {
+              "commandStartedEvent": {
+                "command": {
+                  "delete": "test",
+                  "deletes": [
+                    {
+                      "q": {
+                        "x": {
+                          "$nin": [
+                            24,
+                            34
+                          ]
+                        }
+                      },
+                      "limit": 0
+                    }
+                  ],
+                  "apiVersion": "1",
+                  "apiStrict": true,
+                  "apiDeprecationErrors": {
+                    "$$unsetOrMatches": false
+                  }
+                }
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "deleteOne appends declared API version",
+      "operations": [
+        {
+          "name": "deleteOne",
+          "object": "collection",
+          "arguments": {
+            "filter": {
+              "_id": 7
+            }
+          }
+        }
+      ],
+      "expectEvents": [
+        {
+          "client": "client",
+          "events": [
+            {
+              "commandStartedEvent": {
+                "command": {
+                  "delete": "test",
+                  "deletes": [
+                    {
+                      "q": {
+                        "_id": 7
+                      },
+                      "limit": 1
+                    }
+                  ],
+                  "apiVersion": "1",
+                  "apiStrict": true,
+                  "apiDeprecationErrors": {
+                    "$$unsetOrMatches": false
+                  }
+                }
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "distinct appends declared API version",
+      "operations": [
+        {
+          "name": "distinct",
+          "object": "collection",
+          "arguments": {
+            "fieldName": "x",
+            "filter": {}
+          },
+          "expectError": {
+            "isError": true,
+            "errorContains": "command distinct is not in API Version 1",
+            "errorCodeName": "APIStrictError"
+          }
+        }
+      ],
+      "expectEvents": [
+        {
+          "client": "client",
+          "events": [
+            {
+              "commandStartedEvent": {
+                "command": {
+                  "distinct": "test",
+                  "key": "x",
+                  "apiVersion": "1",
+                  "apiStrict": true,
+                  "apiDeprecationErrors": {
+                    "$$unsetOrMatches": false
+                  }
+                }
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "estimatedDocumentCount appends declared API version",
+      "skipReason": "DRIVERS-1561 collStats is not in API version 1",
```

Comment:
This has been fixed upstream, but the change is not available on all platforms yet. I'll remove the skip once I've confirmed that all our tests are stable with it.

Reply:
FWIW, in the commits where we sync tests I find it's helpful to link to the commit in the specs repo. See: https://github.com/mongodb/mongo-php-library/commit/10b578ffd5ce67e558e92ce14fb7a799820c12d0

Just noticed that I missed mentioning this in all of your recent PRs that updated tests :slightly_smiling_face:

Reply:
Oh, that's a good point. I can amend the commit to note the spec hash.